nimble/host/src/ble_l2cap_coc.c

```diff
@@ -582,7 +597,12 @@ ble_l2cap_coc_recv_ready(struct ble_l2cap_chan *chan, struct os_mbuf *sdu_rx)
         return BLE_HS_EINVAL;
     }
 
-    chan->coc_rx.sdu = sdu_rx;
```

Comment:
What if user call this function more time than BLE_L2CAP_SDU_BUFF_CNT ?

Before we just overwrite chan->coc_rx.sdu which was not great as well.

Reply:
fixed